README.md

```diff
@@ -1,193 +1,266 @@
 for sensitivity analysis, parameter estimation, machine learning applications,
 etc).
 
 ## Breaking changes and new features
 
-**NOTE:** version 14 is a breaking release, prompted by the release of ModelingToolkit.jl version 9. This caused several breaking changes in how Catalyst models are represented and interfaced with.
+**NOTE:** Version 14 is a breaking release, prompted by the release of ModelingToolkit.jl version 9. 
+This caused several breaking changes in how Catalyst models are represented and interfaced with.
 
-Breaking changes and new functionality are summarized in the
-[HISTORY.md](HISTORY.md) file.
+Breaking changes and new functionality are summarized in the [HISTORY.md](HISTORY.md) file. 
+This also includes a special migration guide for version 14.
 
 ## Tutorials and documentation
 
-The latest tutorials and information on using the package are available in the [stable
+The latest tutorials and information on using Catalyst are available in the [stable
 documentation](https://docs.sciml.ai/Catalyst/stable/). The [in-development
 documentation](https://docs.sciml.ai/Catalyst/dev/) describes unreleased features in
 the current master branch.
 
-Several Youtube video tutorials and overviews are also available, but note these use older 
-Catalyst versions with slightly different notation (for example, in building reaction networks):
-- From JuliaCon 2023: A short 15 minute overview of Catalyst as of version 13 is
-available in the talk [Catalyst.jl, Modeling Chemical Reaction Networks](https://www.youtube.com/watch?v=yreW94n98eM&ab_channel=TheJuliaProgrammingLanguage).
-- From JuliaCon 2022: A three hour tutorial workshop overviewing how to use
-  Catalyst and its more advanced features as of version 12.1. [Workshop
-  video](https://youtu.be/tVfxT09AtWQ), [Workshop Pluto.jl
-  Notebooks](https://github.com/SciML/JuliaCon2022_Catalyst_Workshop). 
-- From SIAM CSE 2021: A short 15 minute overview of Catalyst as of version 6 is
-available in the talk [Modeling Biochemical Systems with
-Catalyst.jl](https://www.youtube.com/watch?v=5p1PJE5A5Jw).
-- From JuliaCon 2018: A short 13 minute overview of Catalyst when it was known
-  as DiffEqBiological in older versions is available in the talk [Efficient
-  Modelling of Biochemical Reaction
-  Networks](https://www.youtube.com/watch?v=s1e72k5XD6s)
-
-Finally, an overview of the package and its features (as of version 13) can also be found in its corresponding research paper, [Catalyst: Fast and flexible modeling of reaction networks](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011530).
+An overview of the package, its features, and comparative benchmarking (as of version 13) can also 
+be found in its corresponding research paper, [Catalyst: Fast and flexible modeling of reaction networks](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011530).
 
 ## Features
```

Comment:
Unfortunately, pretty much all of this is marked as changed. This is mainly due to:
- A large part of it (not all) has seen some modifications.
- Quite a few things were moved around (as we now divide stuff into Catalyst features, features of Catalyst composing with other packages, and features of packages building on Catalyst). 

I.e. "[Graphviz](https://graphviz.org/) can be used to generate and visualize reaction network graphs. (Reusing the Graphviz interface created in [Catlab.jl](https://algebraicjulia.github.io/Catlab.jl/stable/).)" wasn't really changed, but it got moved to the second section.